flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -621,15 +621,21 @@ static void process_transmitter_events(ManualControlCommandData * cmd, ManualCon
 	{
 		set_armed_if_changed(FLIGHTSTATUS_ARMED_DISARMED);
 
+		static bool last_arm = false;
```

Comment:
could we move this to the scoping at 628 to make it clear this only alters the behavior of switch arming?